shared/helpers/abi.js

```diff
@@ -0,0 +1,27 @@
+const abi = require('web3-eth-abi')
+
+abiUtil = {
+
+  stringifyParams: (params) => {
+    return params.map(p => {
+      if (Array.isArray(p)) {
+        return abiUtil.stringifyParams(p)
+      }
+      else {
+        if (typeof(p) === 'boolean') {
+          return p
+        }
+        else {
+          return `${p}`
+        }
+      }
+    })
+  },
+
+  encodeFunctionCall: (funcSignature, types, params) => {
+    params = abiUtil.stringifyParams(params)
+    return abi.encodeFunctionSignature(funcSignature) + abi.encodeParameters(types, params).replace('0x', '');
+  }
+}
+
+module.exports = abiUtil
```

Comment:
Let's inline `abiUtil` to be consistent with the rest of the modules here (see other module exports for reference).